### DISCUSSION TITLE
star catalog format

### DISCUSSION THREAD

**Discussion by sronayette** [Stellarium/stellarium#1603] [Q&A]
Hello all,

for an astrometry project of mine (calibration of telescope images - I know other software already do it, but I am having fun writing my own), I was interested in extracting stars position from the Stellarium catalogs (although, I know I can find star catalog elsewhere, I found it convenient, and potentially easier, to use the Stellarium catalog on my computer). I found the wiki page (also appendix C in the user's guide) explaining the catalog format. I can successfully read the data records (and retrieve the star ID, magnitude, parallax, and so on...) but I am still having difficulties to understand the star position. I can read the "position of the star relative to the central point in the star's zone, in axis 1|2", but I am not sure what to do with this number. Are they milliarcsec, or something else? And how can I compute the central point of each zone? Any help on this subject would be very appreciated.

By the way, I have spotted a few mistakes in the catalog format description:
1) The offsets in table C.4 (Data Record type 0) are wrong, starting at the b_v field
2) Still in table C.4, the parallax of the star is retrieved by dividing by 100000 (to get it in arcsec), not 10000
3) In table C.3, the "n" is not the same "n" as calculated in C.1. If there are n zones, the offset of the nth zone is 4.(n-1). Ok, not really a mistake, but can be confusing...

Thanks!

Samuel

**Comment by gzotti** [comment]
Please see the open source code. We have to do the same.
Thanks for the typo hints.

**Reply by sronayette** [reply]
ah ok. Didn't even think about doing that... Thanks.

**Reply by sronayette** [reply]
After a first look, extracting the information I need from this source code is probably way too complicated for me... too bad. Doing reverse engineering on the data is probably an easier solution for me. I'll try a little bit harder.
